provider/ec2/environ_test.go

```diff
@@ -159,11 +146,20 @@ func (*Suite) TestPortsToIPPerms(c *gc.C) {
 			ToPort:    120,
 			SourceIPs: []string{"0.0.0.0/0"},
 		}},
+	}, {
+		about: "source ranges",
+		ports: []network.IngressRule{network.NewIngressRule("tcp", 80, 82, "192.168.1.0/24", "0.0.0.0/0")},
```

Comment:
I would *not* include 0.0.0.0/0 in the acceptable list. 
a) It makes no sense to "allow from everyone" and also "allow from 192...
b) It helps ensure we don't accidentally always include the defaultAllowFromCIDR